cmd/feed-ingress/main.go

```diff
@@ -165,7 +166,7 @@ func main() {
 		Updaters:                updaters,
 		DefaultAllow:            ingressAllow,
 		DefaultStripPath:        ingressStripPath,
-		DefaultBackendKeepAlive: nginxConfig.KeepaliveSeconds,
+		DefaultBackendKeepAlive: nginxBackendKeepaliveSeconds,
```

Comment:
Given we just copy this in without modifying it should we just write directly into the struct like with other nginx config?